dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/database/migration/V9_02_02__DML_Fill_urlaliases.java

```diff
@@ -0,0 +1,290 @@
+import static de.digitalcollections.model.identifiable.entity.EntityType.TOPIC;
+import static de.digitalcollections.model.identifiable.entity.EntityType.VIDEO;
+import static de.digitalcollections.model.identifiable.entity.EntityType.WEBSITE;
+import static de.digitalcollections.model.identifiable.entity.EntityType.WORK;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.openjson.JSONObject;
+import de.digitalcollections.commons.web.SlugGenerator;
+import de.digitalcollections.model.identifiable.IdentifiableType;
+import de.digitalcollections.model.identifiable.alias.UrlAlias;
+import de.digitalcollections.model.identifiable.entity.EntityType;
+import de.digitalcollections.model.identifiable.entity.Website;
+import java.sql.SQLException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.UUID;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.flywaydb.core.internal.jdbc.JdbcTemplate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.jdbc.datasource.SingleConnectionDataSource;
+
+/**
+ * Rollback:
+ *
+ * <ul>
+ *   <li>drop table url_aliases;
+ *   <li>delete from flyway_schema_history where version='9.02.02';
+ * </ul>
+ */
+@SuppressWarnings("checkstyle:typename")
+public class V9_02_02__DML_Fill_urlaliases extends BaseJavaMigration {
+
+  // Cannot be more elegant, since we want to allow null values
+  private static final Map<EntityType, String> ENTITYMIGRATIONTABLES = new HashMap<>();
+
+  static {
+    ENTITYMIGRATIONTABLES.put(AGENT, null);
+    ENTITYMIGRATIONTABLES.put(ARTICLE, "articles");
+    ENTITYMIGRATIONTABLES.put(AUDIO, null);
+    ENTITYMIGRATIONTABLES.put(BOOK, null);
+    ENTITYMIGRATIONTABLES.put(COLLECTION, "collections");
+    ENTITYMIGRATIONTABLES.put(CORPORATE_BODY, "corporatebodies");
+    // FIXME: This must be removed dynamically by the configuration
+    // ENTITYMIGRATIONTABLES.put(DIGITAL_OBJECT, "digitalobjects");
+    ENTITYMIGRATIONTABLES.put(ENTITY, null);
+    ENTITYMIGRATIONTABLES.put(EVENT, null);
+    ENTITYMIGRATIONTABLES.put(EXPRESSION, null);
+    ENTITYMIGRATIONTABLES.put(FAMILY, "familynames");
+    ENTITYMIGRATIONTABLES.put(GEOLOCATION, "geolocations");
+    ENTITYMIGRATIONTABLES.put(HEADWORD_ENTRY, null);
+    ENTITYMIGRATIONTABLES.put(IMAGE, null);
+    ENTITYMIGRATIONTABLES.put(ITEM, "items");
+    ENTITYMIGRATIONTABLES.put(MANIFESTATION, null);
+    ENTITYMIGRATIONTABLES.put(OBJECT_3D, null);
+    ENTITYMIGRATIONTABLES.put(PERSON, "persons");
+    ENTITYMIGRATIONTABLES.put(PLACE, null);
+    ENTITYMIGRATIONTABLES.put(PROJECT, "projects");
+    ENTITYMIGRATIONTABLES.put(TOPIC, "topics");
+    ENTITYMIGRATIONTABLES.put(VIDEO, null);
+    ENTITYMIGRATIONTABLES.put(WEBSITE, "websites");
+    ENTITYMIGRATIONTABLES.put(WORK, "works");
+  }
```

Comment:
Kann man die Map nicht mit `Map.of` erzeugen?

Reply:
Leider ist `Map.of` auf maximal 10 Einträge beschränkt.